yt_dlp/utils/_utils.py

```diff
@@ -1361,6 +1361,23 @@ def brotli(data):
             return data
         return brotli.decompress(data)
 
+    @staticmethod
+    def gz(data):
+        gz = gzip.GzipFile(fileobj=io.BytesIO(data), mode='rb')
+        try:
+            return gz.read()
+        except OSError as original_ioerror:
```

Comment:
Maybe this should be renamed to `os_error` or it should catch `IOError`? Not sure which one it is since the `gzip` docs are not that clear...

Reply:
must be an old relic. IOError has been an alias of OSError since 3.3

https://docs.python.org/3/library/io.html

Reply:
I'm just arguing about naming convention here. I feel `IOError` sounds better, but theoretically it is `OSError`.

Reply:
renamed it to original_oserror here since that's what we are catching.